### PR TITLE
Helper merge steve byerly fork 2

### PR DESCRIPTION
**Pull request by mnowik**

### CODE REVIEW

**Comment by mnowik** [comment]
@SteveByerly Here is your PR. I refactoring some other area of the code.

File: shippo/test/integration/test_integration.py [hunk 1]
```diff
@@ -41,19 +41,36 @@ def test_dns_failure(self):
...
+                self.fail('Set your SHIPPO_API_KEY in your os.envrion')
+        except Exception as inst:
+            self.fail("Test failed with exception %s" % inst)
 
     def test_list_accessors(self):
-        address = shippo.Address.create(**DUMMY_ADDRESS)
+        try:
+            address = shippo.Address.create(**DUMMY_ADDRESS)
+        except shippo.error.AuthenticationError:
+            self.fail('Set your SHIPPO_API_KEY in your os.envrion')
```
Comment (assislucas) [inline shippo/test/integration/test_integration.py]: typo

File: shippo/test/integration/test_integration.py [hunk 2]
```diff
@@ -41,19 +41,36 @@ def test_dns_failure(self):
...
-        self.assertEqual(address.object_state, 'VALID')
-        address = shippo.Address.validate(address.object_id)
-        self.assertEqual(address.object_source, 'VALIDATOR')
+        try:
+            address = shippo.Address.create(**DUMMY_ADDRESS)
+            self.assertEqual(address.object_state, 'VALID')
+            address_validated = shippo.Address.validate(address.object_id)
+            self.assertEqual(address_validated.object_source, 'VALIDATOR')
+        except shippo.error.AuthenticationError:
+                self.fail('Set your SHIPPO_API_KEY in your os.envrion')
```
Comment (assislucas) [inline shippo/test/integration/test_integration.py]: typo

**Comment by assislucas** [comment]
Looks good to me.

**Comment by SteveByerly** [comment]
Cool, thanks! Is there anything you need me to do?

**Comment by sbeidas** [comment]
👍

**Comment by mnowik** [comment]
@SteveByerly all good. Just wanted to give you an update. Thanks again for the PR.

**Pull request merged**